# Exploring the phylogenetic relationships in Abies genus using GBS data.
### Marco Garduño.

### Introduction.

Firs (Abies Miller; a predominantly temperate northern genus) are among the most abundant and less studied taxa, being distributed discontinuously in the temperate and subtropical montane forests of the northern hemisphere of Mesoamerica (i.e. Mexico and northern Central America). Modern firs originated from the divergence of isolated mountain populations of migrating North American taxa. They are represented by eight threatened species, six of which are endemic to Mexico 
They have been grouped into three or two different sections according to the most widely recognized classifications (i.e. Liu, 1971; Farjon and Rushforth, 1989.), which suggests at least two independent expansions from as many ancestors. 

 All phylogenies showed five main clusters that mostly agreed with the currently recognized sections of Abies and with the geographic distribution of species. However, populations of the same species were not monophyletic within this group. The main goal of this project it was to explore de phylogenetic relationships of some of the most relevant firs species in Mexico and northern Central America, using GBS data, and put in contrast with previos infered phylogenetic relationship.   
 
### Materials and methods.

**Sampling of Firs species, DNA extraction and manipulation.**

Needles were collected from nineteen individuals of XX *Abies* species in natural populations (table 1). Samples were obtained by the PhD Students Sebastian XX XX and Gustavo Ibrahim Giles Pérez. Total DNA was extracted from the needles using a cetyltrimethyl ammonium bromide (CTAB) mini-prep protocol (Vázquez-Lobo, 1996), or a DNeasy Plant Mini Kit (Qiagen).

**Sequencing Method and sequence analysis**

Acá hay que pedirle un pocode Ayuda a nuestro buen amigo Gustavo.

Raw sequence reads were clustered *de novo* using Ipyrad (XXXXXX). In this Ensamble i used the mithocondrial genome of *Abies religiosa (GB code: MH612854.1 Abies religiosa voucher DRD45 plastid, complete genome)* in order to eliminate the mitochondrial sequences presented in each sample. Since the Data was demultiplexed previously, I chose those reads with enouhgt high sequencing quality in order to do the next steps (Figure 1	).

Using the loci ouptut data, i select the first 3 000 characters in order to make a phylogenetic reconstruction using RaxML-HPC2 using a GTR+I+G nucleotide substitution model. 
